shared-monitoring/elasticsearch_ecs.tf

```diff
@@ -153,25 +153,61 @@ data "template_file" "userdata_ecs" {
     es_home_dir          = "${local.es_home_dir}"
     es_master_nodes      = "${var.es_master_nodes}"
     es_host_url          = "${aws_route53_record.internal_monitoring_dns.fqdn}:${local.port}"
+    es_block_device      = "${var.es_block_device}"
   }
 }
 
 ############################################
 # CREATE LAUNCH CONFIG FOR EC2 RUNNING SERVICES
 ############################################
 
-module "launch_cfg" {
-  source                      = "git::https://github.com/ministryofjustice/hmpps-terraform-modules.git?ref=master//modules//launch_configuration//blockdevice"
+module "launch_cfg_az1" {
```

Comment:
Could switch to using a count loop to simplify this code and remove duplication/repetition. Example of something similar: https://github.com/ministryofjustice/hmpps-mis-terraform-repo/pull/108